hpx/hpx_init.hpp

```diff
@@ -705,6 +707,18 @@ namespace hpx
     inline int init(util::function_nonser<int(int, char**)> const& f,
         int argc, char** argv, std::vector<std::string> const& cfg,
         hpx::runtime_mode mode = hpx::runtime_mode_default);
+
+/// \cond NOINTERNAL
```

Comment:
What's your rationale not to document the new overloads? Do you intend to deliberately leave them 'undocumented' (in the sense that the user is not supposed to use them)?